### PR TITLE
Allow list for CAS banned #25

### PR DESCRIPTION
**Pull request by IRus**

### CODE REVIEW

File: bot/src/main/kotlin/io/heapy/kotbot/bot/rule/Rules.kt [hunk 1]
```diff
@@ -156,18 +157,23 @@ class DeleteVoiceMessageRule : Rule {
...
-                emit(DeleteMessageAction(message))
-                emit(KickUserAction(message.chatId, userId))
+            if (!casConfiguration.allowlist.contains(userId.toLong())) {
+                val response = client.get<CasResponse>("https://api.cas.chat/check?user_id=$userId")
+                if (response.ok) {
+                    LOGGER.info("User $userId is CAS banned")
+                    emit(DeleteMessageAction(message))
+                    emit(KickUserAction(message.chatId, userId))
+                } else {
+                    LOGGER.info("User $userId is NOT CAS banned")
```
Comment (r4zzz4k) [inline bot/src/main/kotlin/io/heapy/kotbot/bot/rule/Rules.kt:173]: Won't this flood your logs? Or that's okay?

**Review by r4zzz4k — APPROVED** [review]

File: bot/src/main/kotlin/io/heapy/kotbot/bot/rule/Rules.kt (hunk 1 above)
Comment (IRus) [reply bot/src/main/kotlin/io/heapy/kotbot/bot/rule/Rules.kt:173]: Log volume here is pretty small. And even before one log.info happens on each message. I even thinking to increase log volume, and put more metadata about user to logs :)

Comment (IRus) [reply bot/src/main/kotlin/io/heapy/kotbot/bot/rule/Rules.kt:173]: I want to try something small and self-hosted as [loki](https://github.com/grafana/loki) for log storing and processing. Then it's will be even lesser issue

Comment (r4zzz4k) [reply bot/src/main/kotlin/io/heapy/kotbot/bot/rule/Rules.kt:173]: Sure, if that works for you -- no problem :)

**Pull request merged**